eZ/Bundle/EzPublishCoreBundle/Resources/config/templating.yml

```diff
@@ -155,6 +155,55 @@ services:
         tags:
             - { name: twig.extension }
 
+    eZ\Publish\Core\MVC\Symfony\Templating\Twig\Extension\RenderExtension:
+        arguments:
+            $renderStrategy: '@eZ\Publish\SPI\MVC\Templating\RenderStrategy'
+        tags:
+            - { name: twig.extension }
+
+    eZ\Publish\Core\MVC\Symfony\Templating\Twig\Extension\RenderContentExtension:
+        arguments:
+            $renderContentStrategy: '@eZ\Publish\Core\MVC\Symfony\Templating\RenderContentStrategy'
+        tags:
+            - { name: twig.extension }
+
+    eZ\Publish\Core\MVC\Symfony\Templating\RenderStrategy:
+        arguments:
+            $strategies: !tagged_iterator ezplatform.render.strategy
+
+    eZ\Publish\SPI\MVC\Templating\RenderStrategy: '@eZ\Publish\Core\MVC\Symfony\Templating\RenderStrategy'
+
+    eZ\Publish\Core\MVC\Symfony\Templating\RenderContentStrategy:
+        arguments:
+            $defaultMethod: 'inline'
+            $renderMethods: !tagged_iterator ezplatform.render.method
```

Comment:
Wouldn't it be better to use `tagged_locator` instead, since we will always look for a particular render method anyway (using the key)?